eng/versioning/version_client.txt

```diff
@@ -74,6 +74,7 @@ com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-communication-email;1.0.9;1.1.0-beta.1
 com.azure:azure-communication-identity;1.5.1;1.6.0-beta.1
 com.azure:azure-communication-jobrouter;1.1.0;1.1.1
+com.azure:azure-communication-messages;1.0.0
```

Comment:
This should have both current and released versions. 

```suggestion
com.azure:azure-communication-messages;1.0.0;1.0.0
```

Reply:
fixed it. Thank you!